.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -8,7 +8,9 @@ assignees: ''
 ---
 
 ## Documentation
-@trilinos/\<PackageName\>
+@trilinos/\<PackageName\>  
+External users: please refer to the Trilinos package owners page at
+https://github.com/trilinos/Trilinos/wiki/Trilinos-Package-Owners to mention the correct parties because you cannot mention Github teams
```

Comment:
Maybe just delete "@trilinos/<PackageName\>" altogether?

I also find "external users" a bit vague.  How about

"Trilinos organization members, please @ mention the correct package team.
If you are not a member, please refer to https://github.com/trilinos/Trilinos/wiki/Trilinos-Package-Owners to mention the correct parties (because non-members cannot mention Github teams).

Reply:
Good point. It'll be confusing for external users to see that still there. I like the way you phrased it!